src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,8 +1,35 @@
 package core.basesyntax;
 
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.util.List;
+
 public class WorkWithFile {
 
     public void getStatistic(String fromFileName, String toFileName) {
-
+        File baseFile = new File(fromFileName);
+        File reportFile = new File(toFileName);
+        int sumOfSupply = 0;
+        int sumOfBuy = 0;
+        try (BufferedWriter writer = new BufferedWriter(new FileWriter(reportFile, true))) {
+            List<String> data = Files.readAllLines(baseFile.toPath());
+            for (String value : data) {
+                String[] values = value.split(",");
+                if (values[0].equals("supply")) {
+                    sumOfSupply += Integer.parseInt(values[1]);
+                }
+                if (values[0].equals("buy")) {
+                    sumOfBuy += Integer.parseInt(values[1]);
+                }
+            }
```

Comment:
let's move this logic out of this try block
please create some private methods for different steps: reading, calculating, making report, writing